downstream/modules/platform/proc-cli-get-controller-pwd.adoc

```diff
@@ -30,26 +30,27 @@ NAME      HOST/PORT                                              PATH   SERVICES
 example   example-ansible-automation-platform.apps-crc.testing          example-service   http   edge/Redirect   None
 -----
 
-The address for the platform gateway instance is `example-ansible-automation-platform.apps-crc.testing`.
+The address for the {Gateway} instance is `example-ansible-automation-platform.apps-crc.testing`.
 
-== Fetching the platform gateway password
+== Fetching the {Gateway} password
 
-The YAML block for the platform gateway instance in the `AnsibleAutomationPlatform` object assigns values to the _name_ and _admin_user_ keys.
-Use these values in the following command to fetch the password for the platform gateway instance.
+The YAML block for the {Gateway} instance in the `AnsibleAutomationPlatform` object assigns values to the _name_ and _admin_user_ keys.
+
+. Use these values in the following command to fetch the password for the {Gateway} instance.
 
 -----
 oc get secret/<your instance name>-<admin_user>-password -o yaml
 -----
 
-The default value for _admin_user_ is `_admin_`. Modify the command if you changed the admin username in the `AnsibleAutomationPlatform` object.
+. The default value for _admin_user_ is `_admin_`. Modify the command if you changed the admin username in the `AnsibleAutomationPlatform` object.
```

Comment:
This is showing as step 1 in the built doc:
![image](https://github.com/user-attachments/assets/bd02957e-94b2-4a1f-bd7c-6f68e2eea86a)